ruby/private/toolchains/ruby_runtime.bzl

```diff
@@ -101,16 +101,26 @@ def _install_ruby(ctx, ruby):
         shared_library = _relativate(shared_library),
     )
 
-def host_ruby_is_correct_version(ctx, version):
+def get_ruby_info(ctx, interpreter_path):
+    platform = ctx.execute([interpreter_path, "-e", "print RUBY_PLATFORM"]).stdout
+    if platform == "java":
+        ruby_impl = "jruby"
+        ruby_version = ctx.execute([interpreter_path, "-e", "print JRUBY_VERSION"]).stdout
```

Comment:
Is this really the constant you want? JRuby defines `RUBY_VERSION`, too...

Reply:
It does, but RUBY_VERSION points to a different versioning scheme (e.g. jruby-9.2.21.0 is 2.5.8)